tendrl/commons/objects/cluster/atoms/configure_monitoring/__init__.py

```diff
@@ -0,0 +1,101 @@
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons import objects
+from tendrl.commons.objects import AtomExecutionFailedError
+from tendrl.commons.utils.cmd_utils import Command
+
+
+class ConfigureMonitoring(objects.BaseAtom):
+    def __init__(self, *args, **kwargs):
+        super(ConfigureMonitoring, self).__init__(*args, **kwargs)
+
+    def configure_plugin(self, plugin_name, plugin_params):
+        Event(
+            Message(
+                priority="info",
+                publisher=NS.publisher_id,
+                payload={
+                    "message": "Starting configuration of %s on %s with %s"
+                    "as conf parameters" % (
+                        plugin_name,
+                        NS.node_context.fqdn,
+                        plugin_params
+                    )
+                },
+                job_id=self.parameters['job_id'],
+                flow_id=self.parameters['flow_id'],
+            )
+        )
+        cmd_str = "config_manager %s '%s'" % (
+            plugin_name,
+            plugin_params
+        )
+        out, err, rc = Command(cmd_str).run()
+        if not err and rc == 0:
+            Event(
+                Message(
+                    priority="info",
+                    publisher=NS.publisher_id,
+                    payload={
+                        "message": "Configured %s on %s with %s"
+                        "as conf parameters" % (
+                            plugin_name,
+                            NS.node_context.fqdn,
+                            plugin_params
+                        )
+                    },
+                    job_id=self.parameters['job_id'],
+                    flow_id=self.parameters['flow_id'],
+                )
+            )
+            return True
+        else:
+            Event(
+                Message(
+                    priority="error",
+                    publisher=NS.publisher_id,
+                    payload={
+                        "message": "Configuring %s on %s with %s"
+                        "as conf parameters failed.Error %s."
+                        "Return code %s" % (
+                            plugin_name,
+                            NS.node_context.fqdn,
+                            plugin_params,
+                            err,
+                            rc
+                        )
+                    },
+                    job_id=self.parameters['job_id'],
+                    flow_id=self.parameters['flow_id'],
+                )
+            )
+            return False
+
+    def run(self):
+        plugin_config_success = True
+        plugins = NS.config.data.get('plugins', [])
+        plugin_params = {
+            'graphite_host': self.parameters['graphite_host'],
```

Comment:
The only reason I expected this as param is otherwise every node-agent config file needs to have this..
Earlier it used to be configured only in performance-monitoring config file and it used to get passed as param to here. But anyway I'll make it depend on node-agent conf file entry and if that's not present, I'll make it to fall back to defaults(coresident graphite and etcd)..